src/components/search/concept-modal/concept-modal.js

```diff
@@ -98,12 +99,78 @@ export const ConceptModalBody = ({ result }) => {
-    'tranql':       { title: 'TranQL',           icon: <TranQLIcon />,           content: <TranQLTab result={ result } graphs = { graphs } /> }
+    'overview': {
+      title: 'Overview',
+      icon: <OverviewIcon />,
+      content: <OverviewTab result={ result } />,
+      tooltip: <div>
+        The Overview tab provides a brief description of the concept.
+        Hovering over the <InfoCircleOutlined style={{ color: "rgba(0, 0, 0, 0.45)", margin: "0 2px" }} />
+        &nbsp;besides the concept name displays the corresponding ontological
+        identifier, and where applicable, a link to the entry on the ontology&apos;s website.
+      </div>
+    },
+    'studies': {
+      title: studyTitle,
+      icon: <StudiesIcon />,
+      content: <StudiesTab studies={ studies } />,
+      tooltip: <div>
+        The Studies tab displays studies referencing or researching the concept.
+        Studies are grouped into { studySources.length } categories:&nbsp;
+        { studySources.map((source, i) => (
+          <Fragment key={ source }>
+            { i === studySources.length - 1 ? "and " : ""}
+            <span style={{ fontWeight: 500 }}>{ source }</span>
+            { i !== studySources.length - 1 ? ", " : ". " }
+          </Fragment>
+        )) }
+        By default, all studies are shown, but categories can be filtered
+        out by clicking on them. You can also click on studies to view which of their variables are
+        related to the concept.
+      </div>
+    },
+    'cdes': {
+      title: cdeTitle,
+      icon: <CdesIcon />,
+      content: <CdesTab cdes={ cdes } cdeRelatedConcepts={ cdeRelatedConcepts } cdeRelatedStudies={cdeRelatedStudies} loading={ cdesLoading } />,
+      tooltip: <div>
+        The CDEs tab displays{ context.brand === "heal" ? " HEAL-approved" : "" } common data elements (CDEs)
+        associated with the concept. A CDE is a standardized question used across studies and clinical
+        trials with a precisely defined set of permissible responses to ensure consistent data collection.
+        {context.brand === "heal" ? <span>
+          &nbsp;The <a href="https://heal.nih.gov/data/common-data-elements" target="_blank" rel="noopener noreferrer">
+            HEAL CDE program
+          </a> provides further information.
+        </span> : null}
+      </div>
+    },
+    'kgs': {
+      title: 'Knowledge Graphs',
+      icon: <KnowledgeGraphsIcon />, 
+      content: <KnowledgeGraphsTab graphs={ graphs } />,
+      tooltip: <div>
+        The Knowledge Graphs tab displays the <a href="https://www.oxfordsemantic.tech/faqs/what-is-a-knowledge-graph" target="_blank" rel="noopener noreferrer">
+          knowledge graph
+        </a> portion connected to the concept and containing the search terms. This section highlights
+        potential interesting knowledge graph relationships and shows terms (e.g., synonyms) that
```

Comment:
Do we display synonyms on the knowledge graph tab?

Reply:
Oh, I think I see what you mean: we don't display synonyms, but we do display related concepts (with information on what the relation is, e.g. "[Hepatic fibrosis](http://purl.obolibrary.org/obo/HP_0001395) ---[subclass of]--> [liver disorder](http://purl.obolibrary.org/obo/MONDO_0005154)". So I would simplify this to:

```suggestion
        potential interesting knowledge graph relationships and shows related terms.
```

Reply:
I'm not sure, perhaps occasionally but in the vast majority of cases, no. The text here I've been told was reviewed and approved by multiple people, so I would assume it's correct but maybe not.